controls/V-72851.rb

```diff
@@ -104,12 +110,11 @@
   Now reload the server with the new configuration (this just reloads settings
   currently in memory, will not cause an interruption):
 
-  $ sudo su - postgres
   # SYSTEMD SERVER ONLY
-  $ systemctl reload postgresql-9.5
+  $ sudo systemctl reload postgresql-${PG_VER}
```

Comment:
if you update `${PG_VER}` to `#{PG_VER}` it will print out as the real time version number. Not sure if this is useful but we have used it elsewhere in other profiles to make this a bit more automatic in documentation and upkeep.